tests/console/zziplib.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE"s openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: test that uses zip command line tool to regression test.
+# If succeed, the test passes without error.
+#
+# Maintainer: Marcelo Martins <mmartins@suse.cz>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $filezip = "files.zip";
+    select_console "root-console";
+    # create a tmp dir/files to work
+    assert_script_run "mkdir /tmp/zip; cp /usr/share/doc/* /tmp/zip -R";
```

Comment:
@dzedro @martinsmarcelo Could you guys enable the sdk module here? to avoid: https://bugzilla.suse.com/show_bug.cgi?id=1132766? I guess `add_suseconnect_product(get_addon_fullname("sdk))` would do the trick

Reply:
@foursixnine Iooks like it will be not so simple http://10.100.12.155/tests/11619#step/zziplib/12